opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -449,7 +449,11 @@ namespace Opm
             // Run relperm diagnostics if we have more than one phase.
             if (FluidSystem::numActivePhases() > 1) {
                 RelpermDiagnostics diagnostic;
+                if (mpi_size_ > 1)
```

Comment:
For ranks > 0, is it allowed to call switchToGlobalView()? I guess if the diagnosis only runs on rank 0 there might be no risk, but I do not see any "only rank 0" (or output_cout) code here?

Reply:
see 5 lines further up..

Reply:
Thanks (sometimes we are blind).